stl/inc/ximmortalize.h

```diff
@@ -0,0 +1,118 @@
+// ximmortalize.h internal header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _XIMMORTALIZE_H
+#define _XIMMORTALIZE_H
+#include <yvals.h>
+#ifndef _M_CEE_PURE

```

Comment:
Everything but `#include <yvals.h>` is conventionally below `#if _STL_COMPILER_PREPROCESSOR`.